client/security/src/main/java/org/eclipse/kapua/client/security/bean/KapuaPrincipalImpl.java

```diff
@@ -60,10 +60,11 @@ public KapuaPrincipalImpl(AuthResponse authResponse) {
      * @param accountId
      * @param clientId
      */
-    public KapuaPrincipalImpl(KapuaId accountId, String clientId) {
+    public KapuaPrincipalImpl(KapuaId accountId, String name, String clientId) {
```

Comment:
Maybe change this parameter to "username" to distinguish it from the principal "name"?

Reply:
done